tools/configs/uwsgi.ini

```diff
@@ -14,7 +14,7 @@ lazy-apps = true                     ; if this is disabled, we must initialize t
                                      ; and feature_flags.py
 skip-atexit = true                   ; to avoid exception during the on_exit (e.g Pytorch)
 manage-script-name = true
-mount = /=ansible_wisdom.main.wsgi:application
+mount = /=ansible_ai_connect.main.wsgi:application
```

Comment:
Can you please also submit a PR for the Operator.

https://github.com/ansible/ansible-ai-connect-operator/blob/main/roles/model/templates/model.configmap.wisdom-service.yaml.j2#L28

Reply:
https://github.com/ansible/ansible-ai-connect-operator/pull/214